### PR TITLE
Team Popcorn second blog post

### PR DESCRIPTION
**Pull request by sleepypioneer**
Let us know if you need any changes making, huge thanks to you all for an amazing summer ❤️

### CODE REVIEW

File: blog/_posts/2018-09-17-team-popcorn-over-and-out.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+title: Team Popcorn says Tschüss
+layout: post
+created_at: Mon Sept 17 2018
+permalink: blog/2018-07-23-team-popcorn-over-and-out
```
Comment (v-thakkar) [inline blog/_posts/2018-09-17-team-popcorn-over-and-out.md:5]: Hi @sleepypioneer ! Thanks for the PR. Loved the blog post. :+1: 

I just have two minor comments:

1. Permalink and filename should be the same. So fix the date there.

2. If you wish to send a change for Gemfile, you'll need to send a separate PR. Let's keep this PR for blog post only.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-09-17-team-popcorn-over-and-out.md (hunk 1 above)
Comment (sleepypioneer) [reply blog/_posts/2018-09-17-team-popcorn-over-and-out.md:5]: Hi @nerdyvaishali missed this in my list of github notifications will make those changed asap!

Comment (sleepypioneer) [reply blog/_posts/2018-09-17-team-popcorn-over-and-out.md:5]: Hi @nerdyvaishali this was trickier for me than I thought so apologies it took a little longer. I ended up making a new PR for this => #556 I hope that is ok for you and see you Saturday! 🎉

Comment (v-thakkar) [reply blog/_posts/2018-09-17-team-popcorn-over-and-out.md:5]: Hi @sleepypioneer ! Sorry for the late reply. It was nice meeting you.

Actually you can change the same PR by adding revised commits. But anyway the new PR you have sent looks good to me. So, I'm closing this one. Thanks.
